tests/test_trace.py

```diff
@@ -33,39 +33,48 @@
 
 
 class TraceTest(unittest.TestCase):
-    def test_traceon(self):
-
-        z = Zelos(path.join(DATA_DIR, "static_elf_helloworld"))
-        z.plugins.trace.traceon(0x08131B16)
+    def test_trace_inst_start_on_addr(self):
+        z = Zelos(
+            path.join(DATA_DIR, "static_elf_helloworld"),
+            inst_feed="addr=0x08131B16",
+        )
 
         with patch("sys.stdout", new=StringIO()) as stdout:
             z.start()
             self.assertNotIn("[080f5c00]", stdout.getvalue())
             self.assertIn("[08131b16]", stdout.getvalue())
 
-    def test_traceoff(self):
-        z = Zelos(path.join(DATA_DIR, "static_elf_helloworld"), verbosity=1)
-        z.plugins.trace.traceoff(0x08048B73)
+    def test_trace_inst_stop_on_addr(self):
+        z = Zelos(
+            path.join(DATA_DIR, "static_elf_helloworld"),
+            inst_feed="",
```

Comment:
You can accomplish the same thing with `inst_feed=True`, which may be clearer for someone who is unfamiliar with the details of ZML but wants to activate the instruction feed.

Reply:
On the other hand, if changed, may confuse someone who is unfamiliar into thinking that this arg must be a boolean.

Reply:
This is a good point. I think the presence of flags should be indicated by a boolean variable for keyword arguments. Changing test usages to this.